From bdd61777672a1b93cc0f043f96fa489174727a48 Mon Sep 17 00:00:00 2001
From: Suresh Gupta <suresh.gupta@freescale.com>
Date: Sun, 14 Apr 2013 20:25:46 +0530
Subject: [PATCH 717/741] usb: gadget: Turn on cache snooping hardware for
 64-bit Kernel

commit 41882dea7a8f4d041a0f25eea4082ad09996c122 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

Turn on cache snooping hardware for 64-bit Kernel

Signed-off-by: Suresh Gupta <suresh.gupta@freescale.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/usb/gadget/udc/fsl_udc_core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/gadget/udc/fsl_udc_core.c b/drivers/usb/gadget/udc/fsl_udc_core.c
index d594eb183359..0c9996bb6a75 100644
--- a/drivers/usb/gadget/udc/fsl_udc_core.c
+++ b/drivers/usb/gadget/udc/fsl_udc_core.c
@@ -336,7 +336,7 @@ static int dr_controller_setup(struct fsl_udc *udc)
 	}
 #endif
 
-#if defined(CONFIG_PPC32) && !defined(CONFIG_NOT_COHERENT_CACHE)
+#if !defined(CONFIG_NOT_COHERENT_CACHE)
 	/* Turn on cache snooping hardware, since some PowerPC platforms
 	 * wholly rely on hardware to deal with cache coherent. */
 
-- 
2.17.1

